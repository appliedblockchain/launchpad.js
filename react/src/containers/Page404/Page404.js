import React from 'react'
import styles from './style.module.css'

export const Page404 = () => (
  <div className={styles.container}>
    <h1>404 Page not found</h1>
  </div>
)

Page404.propTypes = {}

export default Page404
