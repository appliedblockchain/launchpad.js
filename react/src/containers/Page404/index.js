export { default } from './Page404.js'
