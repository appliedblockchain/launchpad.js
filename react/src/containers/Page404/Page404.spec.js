import React from 'react'
import renderer from 'react-test-renderer'
import { Page404 } from './Page404'

describe('<Page 404 />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Page404 />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
