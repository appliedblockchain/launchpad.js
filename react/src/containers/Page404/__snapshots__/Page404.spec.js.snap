// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Page 404 /> renders correctly 1`] = `
<div
  className="container"
>
  <h1>
    404 Page not found
  </h1>
</div>
`;
